opensearch-operator/controllers/cluster_test.go

```diff
@@ -60,24 +59,22 @@ var _ = Describe("OpensearchCLuster Controller", func() {
 
 	/// ------- Tests logic Check phase -------
 
-	Context("When createing a OpenSearchCluster kind Instance", func() {
+	Context("When creating a OpenSearchCluster kind Instance", func() {
 		It("should create a new opensearch cluster ", func() {
 
 			By("Opensearch cluster")
 			Eventually(func() bool {
 
-				if err := k8sClient.Get(context.Background(), client.ObjectKey{Namespace: ClusterName, Name: "opensearch-yml"}, &cm); err != nil {
```

Comment:
Didn't saw a new ConfigurationController that tests those things that you deleted, am i missing something ?

Reply:
The tests for the tls controller are not a fully-fledged integration test but more a unit test of that one controller. As such I don't need to create the OpenSearchCluster object in kubernetes., I just need to provide it as input to the controller reconcile function and verify what it does. Makes it easier and faster to test features in isolation.

Reply:
With how I changed the behaviour of the configuration controller in the default case it does not create a separate configmap but just relies on the default/demo `opensearch,yml` embedded in the opensearch docker image. As such I removed the test here. At the moment only when the tls controller does something will a configmap be created and that is tested in the `tlsController_test.go`.

Reply:
Just added a separate test case for the configuration controller